src/helpers/sanitize.mjs

```
@@ -1,6 +1,12 @@
 export function sanitizeUrl(str) {
   return str
+    .normalize('NFD') // decompose combined graphemes into base characters and diacritical marks
+    .replace(/[\u0300-\u036f]/g, "") // remove diacritical marks
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)